plugins/fluentd_telemetry_plugin/src/streamer.py

```diff
@@ -54,7 +54,8 @@ def __init__(self, conf_parser):
         self.telem_parser = TelemetryParser(self.config_parser, self.streaming_metrics_mgr,
                                             self.last_streamed_data_sample_per_endpoint,
                                             self.attributes_mngr)
-        self.init_streaming_attributes()
```

Comment:
can we move self.init_streaming_attributes() to be part of the constructor of `TelemetryAttributesManager`?

Reply:
Actually, we cannot, because this method requires an instance of TelemetryParser. However, to create a TelemetryParser instance, we need an instance of TelemetryAttributesManager